topics/proteomics/tutorials/metaquantome-data-creation/tutorial.md

```diff
@@ -0,0 +1,656 @@
+>    - {% icon param-file %} *"Compressed SearchGUI results"*: `searchgui_results` (output of **Search GUI** {% icon tool %})
+>    - *"Specify Advanced PeptideShaker Processing Options"*: `Advanced Processing Options`
+>        - *"The PTM probabilistic score to use for PTM localization"*: `A-score`
+>    - *"Specify Advanced Filtering Options"*: `Advanced Filtering Options`
+>        - *"Maximum Peptide Length"*: `60`
+>    - *"Specify Contact Information for mzIdendML"*: `GalaxyP Project contact (Not suitable for PRIDE submission)`
+>    - In *"Exporting options"*:
+>        - *"Creates a mzIdentML file"*: `Yes`
+>        - *"Compress results into a single zip file"*: `Yes`
+>        - *"Reports to be generated"*: ``
+>
+>
+>
+>    > ### {% icon comment %} Comment
+>    >
+>    >  There are a number of choices for different data files that can be generated using
+ PeptideShaker. A compressed file can be made containing all information needed to view the
+results in the standalone PeptideShaker viewer. A `mzidentML` file can be created that contains
+all peptide sequence matching information and can be utilized by compatible downstream
+software. Other outputs are focused on the inferred proteins identified from the PSMs, as well
+as phosphorylation reports, relevant if a phosphoproteomics experiment has been undertaken.
+More detailed information on peptide inference using SearchGUI and PeptideShaker can be found in 
+our tutorial on [Peptide and Protein ID]({{site.baseurl}}/topics/proteomics/tutorials/protein-id-sg-ps/tutorial.html).
+>    {: .comment}
+>
+{: .hands_on}
+
+
+##  *Removing Contaminants*
+
+> ### {% icon hands_on %} Hands-on: Remove contaminants from PSM report
+> This Select tool is used to remove all the contaminants from the Peptide Spectral Match (PSM) search results.
+>
+> 1. {% tool [Select](Grep1) %} with the following parameters:
+>    - {% icon param-file %} *"Select lines from"*: `output_psm` (PSM report output of **Peptide Shaker** {% icon tool %})
+>    - *"that"*: `NOT Matching`
+>    - *"the pattern"*: `con_`
+>
+>
+>
+>    > ### {% icon comment %} Comment
+>    >
+>    > In Proteomics, contamination is generally detected as peaks in spectra that did not originate 
+from the samples and can be introduced in the sample from a variety of environmental sources or human error. Identification of these 
+contaminants is critical to enable their removal before data analysis, mainly, to maintain the validity of conclusions 
+drawn from statistical analyses. Thus, this selection tool helps us remove the contaminants that were identified in the spectral data. 
+>    {: .comment}
+>
+{: .hands_on}
+
+> ### {% icon question %} Questions
+>
+> 1. Why is removing contaminants important?
+>
+> > ### {% icon solution %} Solution
+> >
+> > 1. Ideally, we would like to remove known contaminants from our samples just to maintain discovering novel proteoforms in our sample.
+> >
+> {: .solution}
+>
+{: .question}
+
+## *Select*
```

Comment:
I would remove this section header because this is not so informative, and the next step is conceptually still part of the contaminant removal